app/src/main/java/com/github/se/eventradar/model/event/Event.kt

```diff
@@ -57,6 +57,7 @@ data class Event(
     map["ticket_name"] = ticket.name
     map["ticket_price"] = ticket.price
     map["ticket_capacity"] = ticket.capacity
+    map["ticket_purchases"] = ticket.purchases
```

Comment:
Thank you! It did cause me problems actually, because addEvent uses toMap() and I was getting errors while retrieving the event after (it would cause my app to crash), but realised it was because ticket_purchases was missing here.